server/modules/routes/cart.routes.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const router= express.Router();
+const pool = require('../modules/pool');
+
+//GET students (Chris work)
+
+router.get('/', (req,res) => {
```

Comment:
router.get does not match filename; propose to change either routes filename or the name in the code

Reply:
Motion passed